webapp-django/crashstats/tools/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from django.conf import settings
+
+from crashstats.crashstats.forms import BaseForm
+from crashstats.supersearch.form_fields import MultipleValueField
+
+
+class NewSignaturesForm(BaseForm):
+
+    end_date = forms.DateField(required=False)
+    start_date = forms.DateField(required=False)
+    not_after = forms.DateField(required=False)
+    product = MultipleValueField(required=False)
+    version = MultipleValueField(required=False)
+
+    def clean_product(self):
+        """By default, we want the product to be set to something, it should
+        not be empty. """
```

Comment:
This should not be done in the form. Defaults are best dealt with in the view. E.g.
```python
product = form.cleaned_data['product'] or settings.DEFAULT_PRODUCT
```